brkt_cli/gcp/gcp_service.py

```
@@ -407,6 +411,20 @@ def get_instance_ip(self, name, zone):
                 pass
         self.log.info("Couldn't find an IP address for this instance.")
 
+    def get_private_ip(self, name, zone):
+        nw = 'networkInterfaces'
+        for i in range(60):
+            time.sleep(5)
+            instance_req = self.compute.instances().get(project=self.project,
+                    zone=zone, instance=name)
+            try:
+                instance = retry(execute_gcp_api_call)(instance_req)
+                if instance[nw][0]['networkIP']:
+                    return instance[nw][0]['networkIP']
+            except:
```

Seems like we won't silently ignore failures. The next line has:
        self.log.info("Couldn't find private IP address for this instance.")

This is the same logic as  get_instance_ip() FWIW